### PR TITLE
Minor specjbb usage cleanup.

### PR DESCRIPTION
**Pull request by jeniferh**

### CODE REVIEW

**Comment by jeniferh** [comment]
Hmm looks like this is failing due to agent/bench-scripts/tests/test-benchmark-clis/test-CL.txt, I am trying to figure out how that test works..

File: agent/bench-scripts/pbench-specjbb2005 [hunk 1]
```diff
@@ -51,9 +51,9 @@ dir=""
 function usage {
 	printf "\tThe following options are available:\n\n"
 	printf -- "\t\t-C str --config=<str>            name of the test config\n"
-	printf -- "\t\t-j str --java_opts=<str>         options passed directly to the JVM\n"
+	printf -- "\t\t-j str --java-opts=<str>         options passed directly to the JVM\n"
 	printf -- "\t\t       --nr-jvms=<int>|node      number of JVMs. if = node, number of JVMs = number of NUMA nodes\n"
-	printf -- "\t\t       --node-pinning=y/n        if y, this script will use numactl to pin JVM to a node (1 per node round-robin)\n"
+	##TODO: printf -- "\t\t       --node-pinning=y/n        if y, this script will use numactl to pin JVM to a node (1 per node round-robin)\n"
```
Comment (portante) [inline agent/bench-scripts/pbench-specjbb2005]: Can you remove this line entirely, and then just file an issue to track adding `--node-pinning` support to `pbench-specjbb2005` instead?

Comment (jeniferh) [reply agent/bench-scripts/pbench-specjbb2005]: Sounds good, I opened issue #1479

**Comment by portante** [comment]
@ndokos, that seem to be intermittent failures.  I just restarted the job to see if it'll pass and then we can merge this.

**Pull request merged**